README.md

```diff
@@ -132,6 +132,7 @@ The goal of an authentication flow is to [set an Okta session cookie on the user
 
   - `username` - User’s non-qualified short-name (e.g. dade.murphy) or unique fully-qualified login (e.g dade.murphy@example.com)
   - `password` - The password of the user
+  - `sendFingerprint` -  Enabling this will send a `X-Device-Fingerprint` header. Defaults to `false`
```

Comment:
it should probably be noted that this is only honored with certain features. We would be able to get a fingerprint without any features but from a usability standpoint, it is only used in conjunction with certain FFs. In fact I think we ignore the header if the flags are not enabled

Reply:
We occasionally include features behind FF (OIDC is an example). As long as sending the header doesn't break anything, I'm not too worried about it.

Reply:
nit: I believe there is an extra space in ```-  Enabling```